src/content/docs/tutorial-create-alerts/manage-alert-quality.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: "Manage your alert quality"
+metaDescription: "How to manage the quality of your alerts with New Relic"
+redirects: 
+  - /docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide/
+  - /docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alert-quality-management-tutorial/
+  - /docs/new-relic-solutions/observability-maturity/aqm-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide
+---
+
+Teams suffer from alert fatigue when experiencing high alert volumes and alerts that aren't aligned to business impact. When this happens,incident responders often assume the alerts are false and have no business impact. In turn, they may start to prioritize easy-to-resolve alerts over others or close unresolved incidents to stay within their response time targets. This almost always results in slower incident response and increased scope and severity when major issues occur.
```

Comment:
`happens,incident` missing space

Reply:
I like the way this ¶ speaks to user goals and needs, but it feels a bit arms-length to me. I suspect that's a carryover from the original OMA doc. Lots of third-person, passive, or conditional language. 

Here's a (unpolished) rewrite for examples of where I think it can get more direct and resonant:

> When a team suffers from too many alerts, or too many false alarms, alert fatigue can set in. As the number of false alarms builds, the fatigue can turn into a negative feedback loop. Overwhelmed incident responders assume the alerts are false, and they prioritize easy-to-resolve alerts or close unresolved incidents to stay within their response time targets. In turn, this means real alerts get missed in the noise—leading to slower incident response, and more severe outages when major issues occur.